aleph/search/query.py

```diff
@@ -205,6 +206,16 @@ def scan(self):
     @classmethod
     def handle(cls, request, limit=None, schema=None, **kwargs):
         parser = SearchQueryParser(request.args, request.authz, limit=limit)
+        # Log the search
+        filters = []
+        for field, values in parser.filters.items():
+            if field not in parser.facet_names:
+                filters.append(field_filter_query(field, values))
+        record_audit.delay(
```

Comment:
There's also prefix searches etc. etc. Would it perhaps be easier to somehow serialise the whole thing? Or at least have a testable `to_dict` method for the parser that generates a full state dump?